democracy/views/section.py

```diff
@@ -52,3 +54,34 @@ def get_queryset(self):
         if not hearing.closed:
             queryset = queryset.exclude(type__identifier=InitialSectionType.CLOSURE_INFO)
         return queryset
+
+
+class RootSectionImageSerializer(SectionImageSerializer):
+    hearing = serializers.SerializerMethodField()
```

Comment:
Hold on, if you only need the hearing ID as the field `hearing`, do
```python
hearing = serializers.CharField(read_only=True, source='hearing_id')
```

and be done with it.